galaxy_ng/app/signals/handlers.py

```diff
@@ -48,6 +50,26 @@ def create_namespace_if_not_present(sender, instance, created, **kwargs):
     Namespace.objects.get_or_create(name=instance.namespace)
 
 
+def set_role_definitions(sender, **kwargs):
```

Comment:
I just realized, you could do
```suggestion
def set_role_definitions(sender, apps=None, verbosity=1, **kwargs):
```
instead of the `kwargs.get`. (Just a matter of preference...)